src/client/AM.PA.MacOs/PersonalAnalytics/Trackers/TrackerManager.swift

```diff
@@ -75,6 +69,32 @@ class TrackerManager {
         return trackers[tracker]!
```

Comment:
This is a little bit scary, and I don't remember why we have this? Can we remove it or think of a better way to access the trackers :)

Reply:
True. At first I wanted to get rid of `getTracker` (its' only used once in the emotion tracker).  Unfortunately, I did not come up with an elegant way to accomplish what was required.

I settled with returning the optional value (see latest commits). With that modification, a tracker can still run into problems if the wrong tracker name is passed as an argument. But at least the tracker can handle that case itself.